src/storage/src/render/debezium.rs

```diff
@@ -62,7 +62,10 @@ where
                                 }
                             };
                             let value = match result.value {
-                                Some(Ok(value)) => value,
+                                Some(Ok((value, 1))) => value,
+                                Some(Ok(_)) => unreachable!(
```

Comment:
where is this checked?

Reply:
in typechecking, we don't allow `ENVELOPE DEBEZIUM` with pg sources